src/routes/safe/components/Apps/index.jsx

```diff
@@ -0,0 +1,210 @@
+// @flow
+import React, { useState, useEffect, useCallback } from 'react'
+import styled from 'styled-components'
+
+import ButtonLink from '../../../../components/layout/ButtonLink'
+import Loader from './ListContentLayout/Loader'
+import sendTransactions from './sendTransactions'
+import { Wrapper, Menu, Content, Footer, Nav } from './ListContentLayout/Layout'
+import List from './ListContentLayout/List'
+
+const StyledIframe = styled.iframe`
+  width: 100%;
+  height: 100%;
+  display: ${props => (props.shouldDisplay ? 'block' : 'none')};
+`
+const operations = {
+  SEND_TRANSACTIONS: 'sendTransactions',
+  GET_TRANSACTIONS: 'getTransactions',
+  ON_SAFE_INFO: 'onSafeInfo',
+  ON_TX_UPDATE: 'onTransactionUpdate',
+}
+
+type Props = {
+  web3: any,
+  safeAddress: String,
```

Comment:
`String` should be lowercase, please check flow docs
https://flow.org/en/docs/types/primitives/

Reply:
done